docs/antctl.md

```diff
@@ -781,6 +781,19 @@ PEER               ASN   STATE
 192.168.77.201:179 65002 Active
 ```
 
+`antctl` agent command `get bgproutes` prints the BGP routes advertised/received to/from the provided BGP peer.
+
+```bash
+# Get the list of advertised bgp routes to the provided bgp peer
+$ antctl get bgproutes --peer 192.168.1.2 --out
+# Get the list of received bgp routes from the provided bgp peer
+$ antctl get bgproutes --peer 192.168.1.3 --in
+# Get the list of all advertised bgp routes
+$ antctl get bgproutes --out
+# Get the list of all received bgp routes
+$ antctl get bgproutes --in
```

Comment:
we do not handle "received routes" in the Antrea BGP implementation, so I think this may be misleading to users
cc @hongliangl

Reply:
> we do not handle "received routes" in the Antrea BGP implementation

Exactly. At least right now, we don't care about the "received routes" from remote peers.  In current implementation of BGPPolicy controller, we only advertise routes, for the "received routes", we do nothing.

Reply:
@hongliangl 
To track, better to create a separate issue to support received routes for  future use-case , as received routes can be important for scenarios involving multiple BGP peers, dynamic route management, or multi-cluster setups, 
as of now for basic CNI and Kubernetes networking, the focus is primarily on what the cluster advertises !